src/testers.jl

```diff
@@ -80,6 +80,10 @@ end
 # Keyword Arguments
    - `output_tangent` tangent to test accumulation of derivatives against
      should be a differential for the output of `f`. Is set automatically if not provided.
+   - `tangent_transforms=TRANSFORMS_TO_ALT_TEST_TANGENTS[]`: a vector of functions that
+      transform the passed argument tangents into multiple tangents that should be tested.
+      Note that the alternative tangents are only passed through the frule, and are not
```

Comment:
maybe a comment here to say that it asserts they don't error.
And somehting about if you need to test their correctness use a seperate `test_frule` with `⊢` and `output_tangent`, passing `tangent_transforms=[]`